addons/mail/static/src/component_hooks/use_update_to_model/use_update_to_model.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module **/
+
+import { useUpdate } from '@mail/component_hooks/use_update/use_update';
+
+const { useComponent } = owl.hooks;
+
+/**
+ * This hook provides support for binding the onMounted/onPatched hooks to the
+ * method of a target record.
+ *
+ * @param {Object} param0
+ * @param {string} param0.localIdPropName Name of the prop of this component
+ *  containing the localId of the target record.
+ * @param {string} param0.methodName Name of the method on the target record.
+ * @param {string} param0.modelName Name of the model of the target record.
+ */
+export function useUpdateToModel({ localIdPropName, methodName, modelName }) {
```

Comment:
`useUpdateToModel` and `useRefToModel` are strange and not elegant, but it goes in the right direction for putting everything in model.

Note that `useUpdateToModel` is assimilable to `afterUpdate`/`onX` dummy fields/lifecycle methods. But considering it's used to overcome DOM API that are not declarative enough (set focus + selection range), it's very much appropriate while using DOM element explicitly.